includes/class-wc-stripe-intent-controller.php

```
@@ -514,8 +513,9 @@ public function update_intent( $intent_id = '', $order_id = null, $save_payment_
 
 			$is_setup_intent = substr( $intent_id, 0, 4 ) === 'seti';
 			if ( ! $is_setup_intent ) {
-				// These parameters are only supported for payment intents.
-				$request['amount']   = WC_Stripe_Helper::get_stripe_amount( $amount, strtolower( $currency ) );
+				// These parameters are only supported for payment intents. Setup
+				// intents carry no amount, so the recalculation is scoped here.
+				$request['amount']   = WC_Stripe_Helper::get_stripe_amount( WC_Stripe_Helper::recalculate_order_total( $order ), strtolower( $currency ) );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'update_intent'. (undefined)

(StaticAccess)